Services/data/devicemodels/scripts/StartMovingDevice-method.js

```diff
@@ -19,8 +18,14 @@
 /*jslint unparam: true*/
 function main(context, previousState, previousProperties) {
 
-    log("Executing JavaScript StartMoving function.");
+    log("Executing JavaScript StartMoving method.");
+
+    var state = {
+        moving: true
```

Comment:
Who actually changes props that show the device is moving?  i.e. does the simulation read this property and change the lat/long incrementally if it's set?

Reply:
Yes, there is a check in prototype-01-state.js:

    // 0.1 miles around some location
    if (state.moving) {
        var coords = varylocation(center_latitude, center_longitude, 0.1);
        state.latitude = coords.latitude;
        state.longitude = coords.longitude;
    }